src/runtime/composables/local/useAuth.ts

```diff
@@ -154,17 +153,20 @@ async function getSession(getSessionOptions?: GetSessionOptions): Promise<Sessio
   return data.value
 }
 
-async function signUp(credentials: Credentials, signInOptions?: SecondarySignInOptions, signUpOptions?: SignUpOptions) {
+async function signUp<T = any>(credentials: Credentials, signInOptions?: SecondarySignInOptions, signUpOptions?: SignUpOptions): Promise<T> {
```

Comment:
```suggestion
async function signUp<T>(credentials: Credentials, signInOptions?: SecondarySignInOptions, signUpOptions?: SignUpOptions): Promise<T> {
```

Can we remove the any typing here? This should result in the response being `unknown` if no type was passed.

Reply:
Sure, we can.